app/src/main/java/com/example/poseexercise/views/fragment/WorkOutFragment.kt

```diff
@@ -210,12 +271,15 @@ class WorkOutFragment : Fragment() {
         // Initialize Exercise Log
         val exerciseLog = ExerciseLog()
         // get the list of plans from database
-        lifecycleScope.launch (Dispatchers.IO) {
+        lifecycleScope.launch(Dispatchers.IO) {
             // get not completed exercise from database using coroutine
-            val notCompletedExercise = withContext(Dispatchers.IO) { homeViewModel.getNotCompletePlans(today)}
-            notCompletedExercise?.forEach {item ->
-                val exercisePlan = ExercisePlan(databaseNameToClassification(item.exercise), item.repeatCount)
-                val existingExercisePlan = databaseExercisePlan.find { it.exerciseName == databaseNameToClassification(item.exercise) }
+            val notCompletedExercise =
```

Comment:
isn't this the same notCompletedPlanList in the onViewCreated?

Reply:
Looks like that. I removed one of them. Thank u Chi!